### PR TITLE
Fixed OpenAPI links

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

**Comment by rssag** [comment]
hm... i'm not sure the openapi spec is in a state where it's a proper replacement for the JSON file we currently have in the documentation. @manasesjesus ?

**Review by manasesjesus — CHANGES_REQUESTED** [review]
this Postman collection JSON is different from the OAS JSON. The first one contains many different examples of all core APIs, plus other examples such as SmartREST. The second one is only the OpenAPI Specification and not a Postman collection; importing/transforming it requires extra steps that we want to automate (those tasks are in my TODO list). Hence, this link should stay as it is, and when the OAS Postman is ready, it'll be included.

**Comment by BeateRixen** [comment]
OK, @JoshuaAdemola then we should close this PR and add a comment to the BH task.

File: content/microservice-sdk/rest-bundle/overview.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Graphical REST clients such as [Postman](https://www.getpostman.com/) are a conv
 
 {{< product-c8y-iot >}} provides numerous online API examples. If you want to make use of them, [download and install Postman](https://www.getpostman.com/). After starting Postman, you can choose to either create an account or click **Take me straight to the app**. Then click the button below and choose the variant of Postman that you have just installed. You may see a browser security prompt asking you if you actually want to run Postman (on Windows "Electron").
```
Comment (manasesjesus) [inline content/microservice-sdk/rest-bundle/overview.md:74]:
```suggestion
{{< product-c8y-iot >}} provides numerous online API examples. If you want to make use of them, [download and install Postman](https://www.getpostman.com/). After starting Postman, you can choose to either create an account or click **Take me straight to the app**.
```

File: content/microservice-sdk/rest-bundle/overview.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ Graphical REST clients such as [Postman](https://www.getpostman.com/) are a conv
...
 
-Import the APIs as a [JSON file](/files/rest/Cumulocity_API.postman_collection.json).
+Import the {{< openapi >}} as a [JSON file](https://www.{{< domain-c8y >}}/api/{{< c8y-current-version >}}/dist/c8y-oas.json).
```
Comment (manasesjesus) [inline content/microservice-sdk/rest-bundle/overview.md:76]:
```suggestion
Download the [API examples](/files/rest/Cumulocity_API.postman_collection.json) and import them as a collection into Postman.
```

**Review by manasesjesus — COMMENTED** [review]
Postman got plenty of updates and that button part is not longer valid. Added suggestions to remove it and update the redaction.

**Review by manasesjesus — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**